package.json

```diff
@@ -18,10 +18,12 @@
     "build-storybook": "storybook build"
   },
   "dependencies": {
+    "@types/dompurify": "^3.0.2",
     "ace-builds": "1.4.12",
     "axios": "1.2.3",
     "core-js": "3.25.1",
     "date-fns": "2.28.0",
+    "dompurify": "^3.0.3",
```

Comment:
Same, remove `^`. We do this so that no new patch versions are included without us explicitly setting them.